lib/replica.rb

```diff
@@ -144,7 +144,8 @@ def api_post(endpoint, key, data)
     return if response.body.empty?
     parsed = Oj.load(response.body)
     raise unless parsed['success']
-    parsed['data']
+    filtered_data = parsed['data'].select { |article| data.include?(article['page_title']) }
+    return filtered_data
```

Comment:
Why is this change to replica.rb included?

Reply:
Not required.
Initially, due to the fact that i was seeing previous vcr response again and again, i thought that the api is not filtering out data with the provided tittle. 
Now when i changed the vcr cassette name for recording response, got to know api is working fine as expected.